test/src/test/java/org/locationtech/geowave/test/basic/GeoWaveBasicCustomCRSRasterIT.java

```diff
@@ -93,9 +93,16 @@ public static void reportTest() {
   @Test
   public void testNoDataMergeStrategy() throws IOException {
     final String coverageName = "testNoDataMergeStrategy";
-    final int tileSize = 64; // 256 fails on bigtable exceeding maximum
-    // size, 128 fails on DynamoDB exceeding
-    // maximum size
+    final int maxCellSize =
+        TestUtils.getTestEnvironment(dataStoreOptions.getType()).getMaxCellSize();
+    final int tileSize;
+    if (maxCellSize <= 64 * 1024) {
+      tileSize = 24;
+    } else {
+      tileSize = 64; // 256 fails on bigtable exceeding maximum size
```

Comment:
We should probably scale the tile size proportional to the max cell size

Reply:
I agree, though that will most likely take more time and require its own PR.